docker-entrypoint.sh

```diff
@@ -47,7 +57,6 @@ if [ "$1" = 'postgres' ]; then
 
 	# look specifically for PG_VERSION, as it is expected in the DB dir
 	if [ ! -s "$PGDATA/PG_VERSION" ]; then
-		file_env 'POSTGRES_INITDB_ARGS'
```

Comment:
This one needs to stay here in case we didn't hit the `[ "$(id -u)" = '0' ]` block for the above `if` (it should be harmless to duplicate it -- the function takes care of making sure multi-invocation is safe).